roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -28,6 +28,7 @@ openshift_logging_mux_journal_source: "{{ openshift_hosted_logging_journal_sourc
 openshift_logging_mux_journal_read_from_head: "{{ openshift_hosted_logging_journal_read_from_head | default('') }}"
 
 openshift_logging_mux_allow_external: False
+openshift_logging_use_mux: "{{ openshift_logging_mux_allow_external | default(False) }}"
```

Comment:
I believe the way ansible works with defaults/main.yml is that if you explicitly set a parameter to a value, it will not use the value set in defaults/main.yml.  So in this case, the user will have set two parameters with conflicting values.  But AFAICT, if `openshift_logging_use_mux=false`, the mux component will not be deployed at all, so `openshift_logging_mux_allow_external` will be ignored.